orca-clouddriver/src/main/java/com/amazon/spinnaker/orca/aws/lambda/CloudDriverProperties.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *  * Copyright 2021 Amazon.com, Inc. or its affiliates.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License")
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *   http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ *
+ */
+
+package com.amazon.spinnaker.orca.aws.lambda;
+
+import com.netflix.spinnaker.orca.clouddriver.OortService;
+import com.netflix.spinnaker.orca.clouddriver.config.CloudDriverConfigurationProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CloudDriverProperties {
```

Comment:
Why do we need this? I get why we needed this for a plugin but I don't believe we need this anymore